protocol/x/clob/keeper/stateful_order_state_test.go

```diff
@@ -1216,12 +795,6 @@ func TestRemoveExpiredStatefulOrdersTimeSlices(t *testing.T) {
 			},
```

Comment:
Can we add a test where multiple orders under the inclusive end blocktime are all removed?

Reply:
Modified the existing test to cover this